app/requirements.txt

```diff
@@ -21,3 +21,4 @@ git+https://github.com/petercuret/keycloak_oidc.git@master
 django-spaghetti-and-meatballs
 django-health-check
 django-redis==4.11.0
+PyJWT<2.0.0
```

Comment:
Deze is niet nodig?

Reply:
PyJWT 2.0.0 heeft een breaking change waardoor djangorestframework-simplejwt faalt. Dit verhelpt het probleem.

Reply:
PyJWT heeft een breaking change waardoor djangorestframework-simplejwt niet meer werkte. Dit verhelpt het probleem.